lib/games_teams_collection.rb

```diff
@@ -55,15 +58,31 @@ def percentage_ties
     ((total_ties.to_f / @games_teams.count) * 100).round(2)
   end
 
-  def max_goals
-    @games_teams.map {|game_team| game_team.goals.to_i}.max
+  def number_of_wins
+    teams = @games_teams.group_by {|game_team| game_team.team_id}
+    teams.values.map do |game|
+      game.count do |game|
+        game.result == "WIN"
+      end
+    end
   end
 
-  def min_goals
-    @games_teams.map {|game_team| game_team.goals.to_i}.min
+  def number_of_losses
+    teams = @games_teams.group_by {|game_team| game_team.team_id}
+    teams.values.map do |game|
+      game.count do |game|
+        game.result == "LOSS"
+      end
+    end
   end
 
   def biggest_blowout
-    max_goals - min_goals
+    difference = []
+    number_of_wins.each do |wins|
+      number_of_losses.each do |losses|
+        difference << wins - losses
+      end
+    end
+    difference.min
   end
```

Comment:
I see what you did here, and that's a smart of way of getting it. I think we still have different interpretations of what the instructions are calling for, though; my take on "highest difference between winner and loser" was that they're looking for the largest difference between the winning team's score and the loser's in a single game, which I think is what a "blowout" usually refers to. I hate to throw a wrench in the cogs. Do you see what I'm saying about the instructions, and/or would you like me to ping the instructors to verify how they mean it?

(If it's the way I'm thinking of it, I think it would take one helper method to find the difference between the high score and low score for a given game_id, a second to grab all the games, and a third to iterate through all the games and run the first method on each to find the highest difference.)

Reply:
(per our conversation - let's get the spec harness working with instructor feedback and then we can figure out what kind of return value it's expecting, and go from there)

Reply:
Got the spec harness working, and it said (when run against the actual data files): 
" expected: 8
    got: -184"
Not sure if that's enough to go by; next step is prob. to ask the instructors, I'll message them in the AM.

Reply:
Got a confirmation from Megan that this method is looking for the highest difference between a winner’s score and the loser’s score in a single game across all the games in the data.